build/simp-utils.spec

```diff
@@ -59,6 +59,12 @@ chmod -R u=rwx,g=rx,o=rx %{buildroot}/usr/local/*bin
 # Post uninstall stuff
 
 %changelog
+* Thu May 02 2019 Jeanne Greulich <jeanne.greulich@onyxpoint.com> - 6.2.0-0
```

Comment:
Since we didn't release the component yet, I would eliminate this changelog content.